config/releases.exs

```diff
@@ -31,6 +31,7 @@ config :nudge, NudgeWeb.Endpoint,
   ],
   secret_key_base: secret_key_base
 
+gigalixir(config: set(POOL_SIZE = 5))
```

Comment:
wait 🤔 
no, this won't work. 

Read this: https://hexdocs.pm/phoenix/ecto.html#repo-configuration
in releases.exs, we need to use environment variables